openhands/llm/llm.py

```diff
@@ -234,10 +227,7 @@ def custom_completion_wait(retry_state):
             before_sleep=log_retry_attempt,
             stop=stop_after_attempt(self.config.num_retries),
             reraise=True,
-            retry=(
-                retry_if_exception_type(self.retry_exceptions)
-                & retry_if_not_exception_type(OperationCancelled)
```

Comment:
Why remove the OperationCancelled exclusion here now?

Reply:
We don't need to exclude specifically, because [`retry_if_exception_type`](https://tenacity.readthedocs.io/en/stable/api.html#tenacity.retry.retry_if_exception_type) will retry only the specified exception types.

`OperationCancelled` is not in `self.retry_exceptions`, so the request won't be retried for it.

Reply:
Ah, right, right. I think Sonnet wanted it to be there "just in case" if that list would be changed.